src/CameraZoomPlugin.cc

```diff
@@ -203,6 +239,7 @@ void CameraZoomPlugin::Impl::InitialiseCamera()
       return;
     }
   }
+  // Todo compute camera sensor width here once rather than every PreUpdate()
```

Comment:
Gazebo use the following annotation for todos:

```c++
//! @todo(user.name@mail.com)
```

I think it's a good practice. It's consistent to search and reminds developers who left stuff to do.

Reply:
I removed this because it doesn't seem possible because all the data objects don't exist yet. Will use that format next time!